kornia/filters/canny.py

```diff
@@ -186,22 +214,22 @@ def __init__(
     def __repr__(self) -> str:
         return (
             self.__class__.__name__ + '('
-            'kernel_size='
-            + str(self.kernel_size)
-            + 'sigma='
-            + str(self.sigma)
-            + 'low_threshold='
-            + str(self.low_threshold)
-            + 'high_threshold='
-            + str(self.high_threshold)
-            + 'hysteresis='
-            + str(self.hysteresis)
-            + 'eps='
-            + str(self.eps)
-            + ')'
+            'kernel_size=' +
+            str(self.kernel_size) +
+            'sigma=' +
+            str(self.sigma) +
+            'low_threshold=' +
+            str(self.low_threshold) +
+            'high_threshold=' +
+            str(self.high_threshold) +
+            'hysteresis=' +
+            str(self.hysteresis) +
+            'eps=' +
+            str(self.eps) +
+            ')'
```

Comment:
@priba try this here https://github.com/kornia/kornia/issues/1053

Reply:
Done. Keep in mind that it will print training=True. from nn.Module